src/app/InteractionModelEngine.cpp

```diff
@@ -61,6 +65,9 @@ CHIP_ERROR InteractionModelEngine::Init(Messaging::ExchangeManager * apExchangeM
 
 void InteractionModelEngine::Shutdown()
 {
+    VerifyOrReturn(mpExchangeMgr != nullptr);
+    VerifyOrReturn(mpFabricTable != nullptr);
```

Comment:
Why do we need mpFabricTable to be non-null to execute a shutdown?

Reply:
There is no good reason for it. I have added it for parity since `InteractionModelEngine::Init` was checking for it to be non-null already.